src/page/WatchedView.jsx

```
@@ -215,71 +290,96 @@ function WatchedView() {
-      {/* Videos List */}
-      <div className="watched-videos">
-        {videos.length === 0 ? (
+      {/* Date filter + Pagination top */}
+      <div className="watched-toolbar">
+        <div className="date-filters">
+          {DATE_FILTERS.map(f => (
+            <button
+              key={f.key}
+              className={`date-filter-btn ${dateFilter === f.key ? 'active' : ''}`}
+              onClick={() => handleDateFilterChange(f.key)}
+            >
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)